src/lib/_common.scss

```diff
@@ -149,6 +149,7 @@ $header-height: 48px;
         align-items: center;
         display: flex;
         flex: 1 1 auto;
```

Comment:
Does it make sense to set the flex-basis to 100%, instead or as well?

Reply:
I tried that but Safari doesn't seem to care about that value either